### ISSUE TITLE
Poco x64-windows-static linking failure

### ISSUE THREAD

**Issue by SergeyLukashevich** [Microsoft/vcpkg#2685]
Run on a clean repository:

```vcpkg install poco:x64-windows-static```

Package: poco:x64-windows-static
Vcpkg version: 0.0.103-2018-01-30-ecaafb9b82252faea6b836ae5b15a8ade2fa2c79

```
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utf8_table1 already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utf8_table1_size already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utf8_table2 already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utf8_table3 already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utf8_table4 already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utt_names already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utt already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_utt_size already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_OP_lengths already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_hspace_list already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_vspace_list already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_ucp_gentype already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
pcre.lib(pcre_tables.c.obj) : error LNK2005: _pcre_ucp_gbtable already defined in PocoFoundationmd.lib(pcre_tables.c.obj)
```

[output.txt](https://github.com/Microsoft/vcpkg/files/1677298/output.txt)
[install-x64-windows-static-rel-out.log](https://github.com/Microsoft/vcpkg/files/1677299/install-x64-windows-static-rel-out.log)

**Comment by SergeyLukashevich** [comment]
After small investigation I found that issue appears somewhere after 30f74216028193e619826809897085a9f536a607 commit. Hope it helps.

**Comment by ras0219-msft** [comment]
Thanks for posting the issue and the digging! It is fixed in master with package version 1.8.1-1.
